app/src/features/rules/components/RulesSidebar/RulesCreateModeSidebar/RulesCreateModeSidebar.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { RuleSelectionList } from "features/rules/screens/rulesList/components/RulesList/components";
+import BotIcon from "assets/icons/bot.svg?react";
+import { RQButton } from "lib/design-system/components";
+import { trackRuleTypeSwitched } from "modules/analytics/events/common/rules";
+import { trackAskAIClicked } from "features/requestBot";
+import { useDispatch } from "react-redux";
+import { actions } from "store";
+import "./RulesCreateModeSidebar.scss";
+
+export const RulesCreateModeSidebar: React.FC = () => {
+  const dispatch = useDispatch();
+
+  return (
+    <div className="rules-create-mode-sidebar-container">
+      <RQButton
+        className="ask-ai-btn"
+        onClick={() => {
+          trackAskAIClicked("rule_sidebar");
+          dispatch(actions.updateRequestBot({ isActive: true, modelType: "ruleTypes" }));
+        }}
+      >
+        <div className="ask-ai-btn-content">
+          <BotIcon />
+          Ask AI for the right rule
+        </div>
+      </RQButton>
+
+      <RuleSelectionList
+        source="rule_sidebar"
+        premiumIconSource="rule_sidebar"
```

Comment:
Why 2 source props with the same value? Can this be achieved using single `source` prop?

Reply:
No, premiumIconSource can be different